Gds/src/fprime_gds/executables/fprime_cli.py

```diff
@@ -63,7 +64,8 @@ def add_retrieval_arguments(parser: argparse.ArgumentParser):
         "--timeout",
         type=float,
         default=0.0,
-        help="wait at most SECONDS seconds for a single new message, then exit (defaults to listening until the user exits via CTRL+C, and logging all messages)",
+        help="wait at most SECONDS seconds for a single new message, then exit (defaults to"
+             "listening until the user exits via CTRL+C, and logging all messages)",
```

Comment:
Same as above. I prefer to not separate the strings into two quotes on separate line. Might be better to revert it back to what they were.